broker/broker.go

```diff
@@ -38,9 +40,10 @@ func Start(metas []model.PluginMeta) {
 	if ok, status := storage.SetStatus(model.StatusStarted); !ok {
 		m := fmt.Sprintf("Starting broker in state '%s' is not allowed", status)
 		pubFailed(m)
-		log.Fatal(m)
+		printError(m)
```

Comment:
Would be better to use term `fatal` in the name to show ppl familiar to Golang specific naming that it is not just logging but also interrupting thread.